Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocMultilineTextValue.css

```diff
@@ -1,15 +1,27 @@
 /* Style applied to the BocMultilineTextValue. */
 
+:root
+{
+  --min-height: 4.5em;
+  --mozilla-offset: 0.1em;
+}
+
 span.bocMultilineTextValue
 {
   width: 15em;
-  min-height: 4.5em;
+  min-height: var(--min-height);
   position: relative;
   display: inline-block;
   vertical-align: middle;
   overflow: hidden;
```

Comment:
When optimzing and there is time, we should check if overflow is still needed. We need to check behavior with long readonly-texts when dropping overflow. Or, possibly, keep it for the readonly-view.